packages/core/src/demo/status-bot.js

```diff
@@ -71,7 +71,7 @@ if (!message) {
 
     const text = message || `I am posting from ${name} v${version} (Build #${process.env.TRAVIS_BUILD_NUMBER}). 🌞`;
     const payload = account.service.conversation.createText(text).build();
-    conversationIds.forEach(async conversationId => await account.service.conversation.send(conversationId, payload));
+    conversationIds.forEach(async conversationId => account.service.conversation.send(conversationId, payload));
```

Comment:
Why did you remove `await` here?

Reply:
Redundant use of `await` on a return

Reply:
`await` on a return isn't always redundant: https://jakearchibald.com/2017/await-vs-return-vs-return-await/

Reply:
But in this case it seems redundant, from the article:

> _Outside of try/catch blocks, return await is redundant. There's even an ESLint rule to detect it, but it allows it in try/catch._

Reply:
and according to ESLint..

Reply:
Ok! It's just a demo anyway :)